apps/system/js/popup_window.js

```diff
@@ -21,6 +21,19 @@
     };
 
     AppWindow.call(this, configs);
+
+    // Replicate the theme color from the parent window.
+    // See http://bugzil.la/1132418
+    if (this.rearWindow) {
+      this.themeColor = this.rearWindow.themeColor;
+      this.element.classList.toggle('light',
+        this.rearWindow.element.classList.contains('light'));
+
+      if (this.statusbar) {
```

Comment:
looks like this is the only place where we style `statusbar.titleBar` directly, I have a bad feeling about this.
can you explain a little more?